df_chat/models.py

```diff
@@ -58,6 +73,65 @@ class ChatMessage(TimeStampedModel):
 
     created_by = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     message = models.TextField(null=False, blank=False)
+    message_type = models.IntegerField(
+        choices=MessageType.choices, default=MessageType.regular_message
+    )
+    related_message = models.ForeignKey(
```

Comment:
should be parent

Reply:
Renamed in #47 and rebased here as well.